tests/console/krb5.pm

```diff
@@ -20,16 +20,18 @@ use strict;
 use serial_terminal;
 use warnings;
 use testapi;
-use serial_terminal 'select_serial_terminal';
+use serial_terminal qw(select_serial_terminal login);
 
 sub logout_and_verify_shell_availability {
     script_run 'logout', 0;
     # verify shell is ready with simple command
     wait_serial(serial_term_prompt(), undef, 0, no_regex => 1);
     # re-connect serial console on aarch64, see poo#157960
     if (is_aarch64) {
-        sleep 3;
+        select_console 'root-console';
+        systemctl 'restart serial-getty@hvc0.service';
         select_serial_terminal;
+        login('root', '#');
```

Comment:
`select_serial_terminal` will do login if you do `reset_consoles` before
That is generally used and would be IMO better.